package.json

```diff
@@ -53,14 +53,13 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint-staged"
+      "pre-commit": "tsc --noEmit && npm run lint-staged"
```

Comment:
`tsc --noEmit $filename` ignore `tsconfig.json` and other files, So we have to run tsc on the whole project.

Or maybe we should add this in ci? @quisquous

Reply:
From TypeScript official:
```bash
# Transpile just the index.ts with the compiler defaults
tsc index.ts
```